demo/README.md

```diff
@@ -0,0 +1,40 @@
+# DataYoga Demo
+
+## Overview
+
+The demo compose file starts the Datayoga `http.receiver` job configured to write incoming records into PostgreSQL database. 
+Generator service is used to generate random records and send them to the receiver.
+
+The demo also starts the exporter, prometheus, and grafana services.
```

Comment:
add link to Grafana (https://grafana.com/)

Reply:
✅